guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -0,0 +1,26 @@
+[id="synchronizing-templates-with-an-existing-repository_{context}"]
+= Synchronizing Templates with an Existing Repository
+
+You can use existing repositories to import and export templates from your {ProjectServer}.
+
+.Procedure
+. Configure a version control system that uses SSH authorization, for example gitosis, gitolite, or git daemon.
+. Configure the TemplateSync plug-in settings on a *TemplateSync* tab.
+.. Change the *Branch* setting to match the target branch on a Git server.
+.. Change the *Repo* setting to match the Git repository.
+For example, for the repository located in `git@git.example.com/templates.git` set the setting into `ssh://git@git.example.com/templates.git`.
```

Comment:
I'm not sure `ssh://` is really needed. AFAIK the ruby-git library that we use doesn't care and handles URLs like the regular git client.

Reply:
Removed.